moveit_ros/moveit_servo/config/panda_simulated_config.yaml

```diff
@@ -26,8 +27,9 @@ publish_joint_velocities: true
 publish_joint_accelerations: false
 
 ## Plugins for smoothing outgoing commands
-use_smoothing: true
-smoothing_filter_plugin_name: "online_signal_smoothing::ButterworthFilterPlugin"
+use_smoothing: false
+smoothing_filter_plugin_name: "online_signal_smoothing::ButterworthFilter"
+joint_acceleration_limit: 15.0
```

Comment:
Why this `joint_acceleration_limit` vs. using the actual moveit params for max joint accelerations?

And I think it's also not reasonable for the "platform agnostic" case to have a single acceleration limit for all possible joints. Especially in the prismatic + revolute combination case.

I think the "correct" thing to do is launching servo with a different `joint_limits.yaml` parameter file than the move group, if you want to use different values.

Reply:
^agree with that. Let's not add a new parameter.

The big-picture plan is to add a Ruckig smoothing plugin that would take care of acceleration limits and replace the ButterworthFilter.

Reply:
This is not a limit imposed by the hardware. The acceleration limit here acts as a smoother. I think it is important to distinguish between desired acceleration/deacceleration and hardware limit values. You rarely want to go at the hardware limits unless you are about to crash. The name is probably not clear enough.

Getting Ruckig smoothing integrated sounds like a good next step. In the meantime, we need a mechanism to enhance servo's acceleration/deacceleration for trajectory interpolation. The current implementation introduces overshoot when combined with trajectory interpolation.

Reply:
>  You rarely want to go at the hardware limits unless you are about to crash.

The same is true for URDF joint limits. Usually they dictate "max desired acceleration for planned motions". Which I think is why @AndyZe and I are taking this stance.

Reply:
Does the URDF even specify acceleration? It has a field for effort, but that is N-m not m/s^2.

Reply:
It sadly does not... it's why MoveIt has its own joint limits file that gets parsed in the config parser:

https://github.com/ros-planning/moveit_resources/blob/ros2/panda_moveit_config/config/joint_limits.yaml

Reply:
I added parsing to extract values from Moveit's internal representation. Additionally, I added a linear scalar parameter for acceleration limits.

Reply:
Gotcha, then this line should now be removed from this file.